### ISSUE TITLE
ACM ethics code is updated and includes stuff we should have

### ISSUE THREAD

**Issue by pimmen** [Widdershin/programmers-oath#124]
The Association for Computing Machinery's Committee on Professional Ethics recently updated their guidelines for being an ethical programmer, after more than 25 years. They described their reasoning in [this Scientific American article](https://www.scientificamerican.com/article/computer-programmers-get-new-tech-ethics-code/). It mentions stuff I've already raised, like not writing software that disenfranchises people through discrimination, but also other stuff like such as being honest about whether or not you believe you can actually accomplish the task with your skills and competence when you take on new work and that we should foster an environment where technical skills are shared amongst us.

There's lots of good stuff in the [ethics code ](https://ethics.acm.org/)in my opinion, what do you guys think?

**Comment by aschrijver** [comment]
Yes, fully agree. I just added ACM code of ethics to [awesome-humane-tech](https://github.com/engagingspaces/awesome-humane-tech) and as a PR to the README as URL in a new Related Work section (see: https://github.com/Widdershin/programmers-oath/pull/127 )

**Comment by aschrijver** [comment]
There is a really cool (but also disturbing) article today in The Guardian: [Franken-algorithms: the deadly consequences of unpredictable code ](https://www.theguardian.com/technology/2018/aug/29/coding-algorithms-frankenalgos-program-danger) that mentions this code of ethics.

I recommend reading the article, and especially stuff in the solutions space are interesting with regards to this repo:

Paraphrasing (emphasis mine):

>Toby Walsh, a professor of artificial intelligence at the University of New South Wales believes **ethics to be the new frontier in tech**, foreseeing “a golden age for philosophy” – a view with which Eugene Spafford of Purdue University, a cybersecurity expert, concurs.
>
>“And we will eventually give up writing algorithms altogether,” Walsh continues, “because the machines will be able to do it far better than we ever could. Software engineering is in that sense perhaps a dying profession. It’s going to be taken over by machines that will be far better at doing it than we are.”
>
>More practically, Spafford, the software security expert, advises making tech companies responsible for the actions of their products, whether specific lines of rogue code – or proof of negligence in relation to them – can be identified or not. He notes that the venerable Association for Computing Machinery has updated its code of ethics along the lines of medicine’s Hippocratic oath, to instruct computing professionals to do no harm and consider the wider impacts of their work.
